dashboards/Pingdom/dashboard.json

```diff
@@ -0,0 +1,505 @@
+													},
+													"signature": ""
+												}
+											},
+											"display": {
+												"range": "zero-fit",
+												"height": 200,
+												"label": "auto",
+												"showKey": true
+											}
+										},
+										"title": "Single Response Time",
+										"description": "Requires $checkId in script"
+									},
+									{
+										"_type": "tile/powershell-as-linegraph",
+										"config": {
+											"context": {},
+											"source": {
+												"scriptTimeoutSecs": 54,
+												"scriptIntervalSecs": 56,
+												"runAsIdentity": "Default",
+												"_security": "signing",
+												"dataType": "object",
+												"columnOverrides": {
+													"timestamp": "timestamp",
+													"value": "value",
+													"grouping": "site"
+												},
+												"split": false,
+												"profile": "Pingdom",
+												"script": "## Enter check IDs (can be found in check detail below)\n$checkId1 = ''\n$checkId2 = ''\n\n## Connection Info\n$uri= \"summary.performance/\"\n$check1 = Invoke-RestMethod -Uri ($url+$Uri+$checkId1) -Headers $headers -Method Get\n$check2 = Invoke-RestMethod -Uri ($url+$Uri+$checkId2) -Headers $headers -Method Get\n\n## Foreach loop to add site property for both checks\nforeach ($item1 in $check1) {\n        $item1.summary.hours | Add-Member -MemberType NoteProperty \"site\" -Value \"SquaredUp.com\"\n        }\n\nforeach ($item2 in $check2) {\n        $item2.summary.hours | Add-Member -MemberType NoteProperty \"site\" -Value \"API1\"\n        }\n\n## Parse results and adjust key names to suit SquaredUp's needs\n### Check1\n$check1.summary.hours | select-object @{\n        Name = \"value\";\n        Expression = {$_.avgresponse}\n},\n@{\n        Name = \"timestamp\";\n        Expression = {(Get-Date 01.01.1970) + ([System.TimeSpan]::fromseconds([int]$_.starttime))}\n},\n@{\n        Name = \"site\";\n        Expression = {$_.site}\n}\n\n### Check2\n$check2.summary.hours | select-object @{\n        Name = \"value\";\n        Expression = {$_.avgresponse}\n},\n@{\n        Name = \"timestamp\";\n        Expression = {(Get-Date 01.01.1970) + ([System.TimeSpan]::fromseconds([int]$_.starttime))}\n},\n@{\n        Name = \"site\";\n        Expression = {$_.site}\n}\n\n## Combine to queries\n$combined = @($check1;$check2)\n\n## Output results\n$combined.summary.hours",

```

Comment:
Are the Values here supposed to be hardcoded to SquaredUp.com and API1? 

![image](https://user-images.githubusercontent.com/18680913/130097156-8aedfded-bb62-45c0-b5cd-870f6455ee5d.png)

Reply:
It should be API1 and API2 - these are for the label in SQUP. I've updated the values to "API12 and "API2"